test/cookbooks/osl-matrix-test/recipes/synapse.rb

```diff
@@ -0,0 +1,23 @@
+include_recipe 'osl-docker'
```

Comment:
This should technically not be needed if you're doing this inside the resources.